pxr/usd/usdPhysics/parseDesc.h

```diff
@@ -48,40 +31,40 @@ const float usdPhysicsSentinelLimit = 0.5e38f;
 ///
 struct UsdPhysicsObjectType
 {
-    enum Enum
-    {
-        eUndefined,
+	enum Enum
```

Comment:
I am really sorry for asking for this, as this can be fixed internally (however we have being trying to not make much explicit changes on top of a PR being incorporated). 

Could you please fix tab -> 4 spaces here? (Possible your editor is not mapping tabs to 4 spaces?).

Thanks